IBAnimatable/PanInteractiveAnimator.swift

```diff
@@ -60,7 +60,7 @@ public class PanInteractiveAnimator: InteractiveAnimator {
     progress = min(max(progress, 0), 0.99)
 
     // Finish the transition when pass the threathold
-    let shouldFinishInteractiveTransition =  progress > 0.5 || speed > 1000
+    let shouldFinishInteractiveTransition = progress > 0.5 || speed > 1_000
```

Comment:
That's due to `number_separator`: separate big numbers by "_" to make it easier to read. I'm not a huge fan for 1000, but starting with the next version, we can configure it to warn for number only > 10_000